Scripts/create_pods_and_versions_index.rb

```diff
@@ -2,47 +2,150 @@
 # frozen_string_literal: true
 
 require 'cocoapods'
+require 'parallel'
 
-source = Pod::Source.new('.')
+def parallel?
+  ENV['COCOAPODS_PARALLEL']
+end
+
+def netlify?
+  ENV['NETLIFY']
+end
+
+class UI
```

Comment:
Perhaps move into a separate source file?

Reply:
Have moved the others but I think it makes sense to keep this here

Reply:
ended up moving it